.github/CODEOWNERS

```diff
@@ -82,7 +78,6 @@
 /StRoot/StRefMultCorr                              @nigmatkulov	      @ToshihiroNonaka
```

Comment:
Please replace @ToshihiroNonaka with @zsweger  (zwsweger@ucdavis.edu).

Reply:
Invitation was sent to @zsweger to join the team, once accepted we can add. This PR is about removal